crates/providers/src/anilist/mod.rs

```diff
@@ -684,6 +699,7 @@ async fn media_details(
         title.romaji,
         preferred_language,
     );
+    let status = details.status.unwrap();
```

Comment:
I think it would be more prudent to not unwrap it forcefully here. The return method expects an `Option` anyway.

Reply:
This needs to be unwrapped to convert to a string since this is a MediaStatus enum